komorebi/consts.in

```diff
@@ -0,0 +1,4 @@
+package_name : str = @PACKAGE_NAME@
```

Comment:
I'm not sure about this, now, but this variable seems more likely to be inside `__init__.py`. At some of them, like version.

Reply:
Fixed